Formula/m/magic-wormhole.rb

```diff
@@ -117,13 +118,6 @@ class MagicWormhole < Formula
   resource "spake2" do
     url "https://files.pythonhosted.org/packages/60/0b/bb5eca8e18c38a10b1c207bbe6103df091e5cf7b3e5fdc0efbcad7b85b60/spake2-0.8.tar.gz"
     sha256 "c17a614b29ee4126206e22181f70a406c618d3c6c62ca6d6779bce95e9c926f4"
-
```

Comment:
patch should be remain or altered for
 - https://github.com/warner/python-spake2/pull/18